Merge MorphOS branches

MBeijer opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

## Existing Issue(s)
<!--- If it fixes an open issue, please link to the issue here. -->

capehill commented:
@MBeijer not sure how to reply here, but yeah, I agree. After SDL 2.0.7 "final" is out, I will probably start using this repo.

MBeijer commented:
@capehill Meanwhile, I've set up on my own repo so I can easily merge in new changes from the sdl2-amigaos4 repo master branch over to this one. =)

MBeijer commented:
@capehill I suggest joining up in our AmigaPorts Slack: https://join.slack.com/t/amigaports/shared_invite/enQtMzE2NjE1MDg0OTkzLTYyYmRmZjliMTJhMGExZDUxMmFhZmIwMzA4NWNkZDIzMzcxMzE4OWFhY2UyMDZhODAzNDllMTU1NWFmNTdiZDU

It'll be easier to discuss things in realtime and don't mess upp issues and pull requests like this.

I've set it up to post all updates from this, sdl2-amigaos4 and libSDL12 in there as well.

Either you just use it in your browser or download the application =)

I understand if you don't feel like doing it, but it sure would help organize everything.

BeWorld2018 committed:
MorphOS: add last SDL2

BeWorld2018 committed:
Add missing makefile

BeWorld2018 committed:
Update .gitignore

BeWorld2018 committed:
Little Fix

BeWorld2018 committed:
MorphOS: Fix SDL_CreateCursor

- fix pointer from surface
- add 2021 year in copyright

BeWorld2018 committed:
Morphos: Another little fix with pointer

BeWorld2018 committed:
Merge branch 'main' into MorphOS

BeWorld2018 committed:
Fix: update

Fix Read mouse.conf if DoubleClick/DoubleClickM is not present

BeWorld2018 committed:
MorphOS: cleanup

CleanUp joystick code + fix index_player detection
Add Xbox360 controller
Fix AMIGA_GetDoubleClickTimeInMillis

BeWorld2018 committed:
Fix right button in fullscreen

Find bug on OpenTTD/Fullscreen right click to move map

BeWorld2018 committed:
MorphOS : add altivec to SDL_BMP

BeWorld2018 committed:
MorphOS : add Low Priority in menu

BeWorld2018 committed:
MorphOS: somes cleanup

Reverse : fix index_player detection in joystick...

BeWorld2018 committed:
Merge branch 'main' into MorphOS

BeWorld2018 committed:
MorphOS : add SDL_CopyAndSwap16/32 function

altivec byteswap available into library

BeWorld2018 committed:
MorphOS: fix

Missing SDL_CopyAndSwap16 and SDL_CopyAndSwap32 in SDL2 headers

BeWorld2018 committed:
Merge pull request #1 from BeWorld2018/morphos_swapbyte

Morphos swapbyte

BeWorld2018 committed:
Add "About System"

Add more informations about joystick

BeWorld2018 committed:
Merge branch 'main' into MorphOS

BeWorld2018 committed:
MorphOS: add SDL_GetAudioDeviceSpec STUB

BeWorld2018 committed:
MorphOS: fix joystick init

BeWorld2018 committed:
MorphOS: fix crash when AHI is already opend

BeWorld2018 committed:
MorphOS : fix mouse move

Fix coords when activated window.

BeWorld2018 committed:
Merge pull request #2 from libsdl-org/main

Update Main

BeWorld2018 committed:
Merge branch 'main' into MorphOS

BeWorld2018 committed:
Merge remote-tracking branch 'upstream/main' into main

BeWorld2018 committed:
Merge branch 'main' into MorphOS

BeWorld2018 committed:
MorphOS: add AMIGA_GetWindowBordersSize

BeWorld2018 committed:
Merge remote-tracking branch 'upstream/main' into main

BeWorld2018 committed:
Merge branch 'main' into MorphOS

BeWorld2018 committed:
MorphOS: makefile test

delete nativeamigaos4

BeWorld2018 committed:
MorphOS: changer order SDL_CopyAndSwap function in library

BeWorld2018 committed:
[AMIGA_CreateCursor] Fix cursor position
